actions/post_attachment.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+from six.moves.urllib.parse import urlencode
+import requests
+
+from st2common.runners.base_action import Action
+
+__all__ = [
+    'PostMessageAction'
```

Comment:
It's probably best to rename the class to PostAttachmentAction here and in the class definition, to avoid confusion with the PostMessageAction class in the post_message.py.

Reply:
Agreed, added it back in to both actions and updated config and example to provide that as an option. 

Also, good call on the `PostAttachmentAction`. 👍